ext/mbstring/mbstring.c

```diff
@@ -2967,7 +2970,7 @@ PHP_FUNCTION(mb_convert_kana)
 	size_t optstr_len;
 	zend_string *encname = NULL;
 
-	if (zend_parse_parameters(ZEND_NUM_ARGS(), "s|sS", (char **)&string.val, &string.len, &optstr, &optstr_len, &encname) == FAILURE) {
+	if (zend_parse_parameters(ZEND_NUM_ARGS(), "s|s!S!", (char **)&string.val, &string.len, &optstr, &optstr_len, &encname) == FAILURE) {
```

Comment:
It looks like we can default `$opt = "KV"` here.

Reply:
The default has been changed in the stub, so the `!` should be dropped from `s`.